packages/component-library/components/Spacing/Base.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "~@kaizen/component-library/styles/fonts";
+@import "~@kaizen/component-library/styles/type"; // just for the grid unit
+
+$spacing-list: (
+  ("eighth", $ca-grid/8),
```

Comment:
I made sure the classnames these spits out are somewhat easy to reason about. 
Padding of 0 is `p-0`
Padding of 1/8 is `p-eighth`